hard coded schema name when WebAPI comes up

Hi @chgl ,
I was working on the CDM init container and noticed one issue.  

I am using the schema name  webapi for storing all the webapi database objects and I am using cdm for storing the CDM data.

I have used the above two schema names in my values.yaml and also the webapi.db.schema is set to webapi.

However, I still see the webapi container crashing with the following error.  It looks like it is looking at a schema named OHDSI by default.

I have these for the extraEnv in webapi:

```yaml
  - name: "datasource.cdm.schema"
      value: "cdm"
    - name: "datasource.ohdsi.schema"
      value: "webapi"
    - name: "spring.jpa.properties.hibernate.default_schema"
      value: "webapi"
    - name: "spring.batch.repository.tableprefix"
      value: "webapi.BATCH_"
    - name: "flyway_schemas"
      value: "webapi"
    - name: "flyway.placeholders.ohdsiSchema"
      value: "webapi"
```
```
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flyway' defined in class path resource [org/ohdsi/webapi/FlywayConfig.class]: Invocation of init method failed; nested exception is org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateSqlException: 
Migration V1.0.0.1__schema-create_spring_batch.sql failed
---------------------------------------------------------
SQL State  : 3F000
Error Code : 0
Message    : ERROR: schema "ohdsi" does not exist
  Position: 14
Location   : db/migration/postgresql/V1.0.0.1__schema-create_spring_batch.sql (/var/lib/ohdsi/webapi/WEB-INF/classes/db/migration/postgresql/V1.0.0.1__schema-create_spring_batch.sql)
Line       : 1
Statement  : CREATE TABLE ohdsi.BATCH_JOB_INSTANCE  (
	JOB_INSTANCE_ID BIGINT  NOT NULL PRIMARY KEY ,
	VERSION BIGINT ,
	JOB_NAME VARCHAR(100) NOT NULL,
	JOB_KEY VARCHAR(32) NOT NULL,
	constraint JOB_INST_UN unique (JOB_NAME, JOB_KEY)
) 
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1630)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:297)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1139)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1067)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:835)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:741)
	... 26 more
Caused by: org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateSqlException: 
Migration V1.0.0.1__schema-create_spring_batch.sql failed
---------------------------------------------------------
SQL State  : 3F000
Error Code : 0
Message    : ERROR: schema "ohdsi" does not exist
  Position: 14
Location   : db/migration/postgresql/V1.0.0.1__schema-create_spring_batch.sql (/var/lib/ohdsi/webapi/WEB-INF/classes/db/migration/postgresql/V1.0.0.1__schema-create_spring_batch.sql)
```

--- a/charts/ohdsi/Chart.yaml
+++ b/charts/ohdsi/Chart.yaml
@@ -18,4 +18,4 @@ dependencies:
     version: 10.3.11
     repository: https://charts.bitnami.com/bitnami
     condition: postgresql.enabled
-version: 0.4.14
+version: 0.4.15

--- a/charts/ohdsi/ci/custom-schema-name-values.yaml
+++ b/charts/ohdsi/ci/custom-schema-name-values.yaml
@@ -1,0 +1,3 @@
+webApi:
+  db:
+    schema: webapi

--- a/charts/ohdsi/templates/_helpers.tpl
+++ b/charts/ohdsi/templates/_helpers.tpl
@@ -47,15 +47,15 @@ Get the name of the secret containing the DB password
 {{- define "ohdsi.webapi.db-secret-name" -}}
 {{- $fullname := ( include "ohdsi.fullname" . ) -}}
 {{- if .Values.postgresql.enabled -}}
-{{- if .Values.postgresql.existingSecret -}}
-{{ .Values.postgresql.existingSecret | quote }}
-{{- else -}}
-{{ printf "%s-%s" $fullname "postgresql" }}
-{{- end -}}
+    {{- if .Values.postgresql.existingSecret -}}
+        {{ .Values.postgresql.existingSecret | quote }}
+    {{- else -}}
+        {{ printf "%s-%s" $fullname "postgresql" }}
+    {{- end -}}
 {{- else if .Values.webApi.db.existingSecret -}}
-{{ .Values.webApi.db.existingSecret | quote }}
+    {{ .Values.webApi.db.existingSecret | quote }}
 {{- else -}}
-{{ printf "%s-%s" $fullname "webapi-db-secret" }}
+    {{ printf "%s-%s" $fullname "webapi-db-secret" }}
 {{- end -}}
 {{- end -}}
 
@@ -64,10 +64,10 @@ Create the JDBC URL from the host, port and database name.
 */}}
 {{- define "ohdsi.webapi.jdbcUrl" -}}
 {{- if .Values.postgresql.enabled -}}
-{{- $fullname := ( include "ohdsi.fullname" . ) -}}
-{{- $pgServiceName := ( printf "%s-%s" $fullname "postgresql") -}}
-{{ printf "jdbc:postgresql://%s:%d/%s" $pgServiceName 5432 .Values.postgresql.postgresqlDatabase }}
+    {{- $fullname := ( include "ohdsi.fullname" . ) -}}
+    {{- $pgServiceName := ( printf "%s-%s" $fullname "postgresql") -}}
+    {{ printf "jdbc:postgresql://%s:%d/%s" $pgServiceName 5432 .Values.postgresql.postgresqlDatabase }}
 {{- else -}}
-{{ printf "jdbc:postgresql://%s:%d/%s" .Values.webApi.db.host (int64 .Values.webApi.db.port) .Values.webApi.db.database }}
+    {{ printf "jdbc:postgresql://%s:%d/%s" .Values.webApi.db.host (int64 .Values.webApi.db.port) .Values.webApi.db.database }}
 {{- end -}}
 {{- end -}}

--- a/charts/ohdsi/templates/webapi-deployment.yaml
+++ b/charts/ohdsi/templates/webapi-deployment.yaml
@@ -69,6 +69,8 @@ spec:
               value: {{ $host.host | quote }}
               {{- end }}
             {{- end }}
+            - name: DATASOURCE_DRIVERCLASSNAME
+              value: org.postgresql.Driver
             - name: DATASOURCE_URL
               value: {{ include "ohdsi.webapi.jdbcUrl" $ }}
             - name: DATASOURCE_USERNAME
@@ -86,6 +88,12 @@ spec:
               value: {{ .db.schema }}
             - name: SPRING_JPA_PROPERTIES_HIBERNATE_DEFAULT_SCHEMA
               value: {{ .db.schema }}
+            - name: SPRING_JPA_PROPERTIES_HIBERNATE_DIALECT
+              value: org.hibernate.dialect.PostgreSQLDialect
+            - name: SPRING_BATCH_REPOSITORY_TABLEPREFIX
+              value: {{ .db.schema }}.BATCH_
+            - name: FLYWAY_DATASOURCE_DRIVERCLASSNAME
+              value: org.postgresql.Driver
             - name: FLYWAY_SCHEMAS
               value: {{ .db.schema }}
             - name: FLYWAY_PLACEHOLDERS_OHDSISCHEMA

